util/travis/clang-format-whitelist.txt

```diff
@@ -402,3 +402,7 @@ src/voxelalgorithms.h
 src/voxel.cpp
 src/voxel.h
 src/wieldmesh.cpp
+src/meshgen/extruder.cpp
```

Comment:
I think new files should be well-formatted, why do you exclude them?

You can use
```c++
// clang-format off
// clang-format on
```
around lines which have a special format. (based on [this](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) website)

Reply:
> I think new files should be well-formatted, why do you exclude them?

Because linter is stupid.

> You can use
>
>     // clang-format off
>     // clang-format on

Thanks. That helps.